### ISSUE TITLE
Can WPA3 have any attack methods?

### ISSUE THREAD

**Issue by QianSong1** [v1s1t0r1sh3r3/airgeddon#533]
#### Is your question related to any concrete technology?
Seeking technical leaders to teach attack technology

**Comment by OscarAkaElvis** [comment]
Give it a try to the WPA3 plugin!

**Comment by QianSong1** [comment]
I tracked the channel and attacked the target's 2.4Ghz signal
![image](https://user-images.githubusercontent.com/101644710/228780335-3c2c080d-6006-4aa9-b70f-54067f66591c.png)

And simultaneously tracked the channel and attacked the target 5GHz signal
![image](https://user-images.githubusercontent.com/101644710/228781053-bc29ca16-9603-4b9f-99c9-a3ba76c0abfb.png)

And launched an evil twin attack, creating a fake AP
![image](https://user-images.githubusercontent.com/101644710/228781533-e11f4006-fcd7-44ac-84cf-9f4fae985919.png)

For WPA2 type networks, as long as I wait patiently, most of the victims will surrender, and then I successfully captured the WiFi password.



However....



For a WPA3 type network, I can use my phone to create a hotspot AP with the same name, set the encryption method to WPA2, and then use the handshake packet of the phone's AP to launch an evil twin attack. Then look for the correct password from the historical password file:/tmp/www/ag.attemps.txt, and try connecting to the WPA3 type network. The key is to get stuck in the step where the target network type of DOS is WPA3. Currently, DOS is invalid for WPA3. I wonder if there is any signal jammer that can kill the WPA3 machine. Haha...

**Comment by OscarAkaElvis** [comment]
Yeah, it seems you'll need to wait for now...

On the other hand, based on your screenshots I can see you speak chinese, right? and it seems you have good english level. We are looking for a chinese translator for airgeddon. Are you interested in collaborate with us to add chinese language to airgeddon? The hard part of the work will be done automatically by scripting and the chinese translator will need only to review and fix the strings as needed. If you are interested contact me by Discord, tweeter, IRC, email or any other way. Thanks.

**Comment by QianSong1** [comment]
![image](https://user-images.githubusercontent.com/101644710/228826445-3bae0c9b-633e-44a4-9ccf-e454a53b8816.png)


Hee hee, I used machine translation to complete it. It seems that my English level is very good~~~


![image](https://user-images.githubusercontent.com/101644710/228826843-d4d2142c-f1e8-4bf2-9792-784b3c18584b.png)

**Comment by OscarAkaElvis** [comment]
Sorry then... if you are not good speaking english, then it can't be done. Thanks anyway.

**Comment by OscarAkaElvis** [comment]
Yeah but to do that we still need to find somebody chinese native speaker, also speaking english, basic git knowledge and interested in contribution to community.

**Comment by QianSong1** [comment]
"Yes, this is a technical job. Maybe accidentally, it may be due to merging Git branches or forking the code in the repository, which may damage the airgeddon language environment. Therefore, I can't contribute here. Sorry."~~~

**Comment by ChillVibesMushroom** [comment]
I have a question related to WPA 3 about my router I was trying to setup my router to only accept WPA3 connections and  I I get this far but when I go to connect through kali it asks for a username and password man and idk what that is man it hurts me so bad man over this i was in pain and I was wondering if you can help me figure it out please
![settingupwpadatmexico](https://user-images.githubusercontent.com/99580833/229135517-1258574e-42d5-495f-afaa-64ba3b18d4dd.png)

**Comment by OscarAkaElvis** [comment]
Don't divert the thread. Locking.

If you want to ask random questions, try on Discord channel. Invitation link is in the wiki.
